Gems/Vegetation/Code/Source/AreaSystemComponent.cpp

```diff
@@ -593,7 +593,11 @@ namespace Vegetation
             });
     }
 
-    void AreaSystemComponent::OnSurfaceChanged(const AZ::EntityId& /*entityId*/, const AZ::Aabb& oldBounds, const AZ::Aabb& newBounds)
+    void AreaSystemComponent::OnSurfaceChanged(
+        const AZ::EntityId& /*entityId*/,
```

Comment:
Nit: Since this line is being changed might want to switch to [[maybe_unused]] instead of the commented out param name